app/models/version.rb

```diff
@@ -369,7 +373,7 @@ def sha256_hex
   end
 
   def self._sha256_hex(raw)
-    raw.unpack1("m0").unpack1("H*")
+    raw&.unpack1("m0")&.unpack1("H*")
```

Comment:
Does this belong here? Maybe a separate PR? Looks related to the spec_sha256.

Reply:
I did quick check and it seems this method is used only in `spec256_hex` with guard for nil value already and in GemInfo model (not related tot his change). Reverted.